app/MarketHighlight.tsx

```diff
@@ -0,0 +1,101 @@
+'use client';
+
+import { useQueries } from '@tanstack/react-query';
+import Link from 'next/link';
+import { FixedProductMarketMaker, getMarket } from '@/queries/omen';
+import { OutcomeBar, TokenLogo } from '@/app/components';
+import { formatEtherWithFixedDecimals, remainingTime } from '@/utils';
+import {
+  Carousel,
+  CarouselContent,
+  CarouselItem,
+  CarouselNext,
+  CarouselPrevious,
+  CarouselSelector,
+} from './components/Carousel';
+import Autoplay from 'embla-carousel-autoplay';
+import { highlightedMarketsList } from '@/market-highlight.config';
+import defaultHighlightImage from '@/public/assets/highlights/default.png';
+import Image from 'next/image';
+import { useState } from 'react';
+
+export const MarketHighlight = () => {
```

Comment:
The name should be plural as we are highlighting markets
```suggestion
export const MarketsHighlight = () => {
```